frontend/language/src/nb.json

```diff
@@ -134,10 +135,9 @@
   "dashboard.data_modelling_format_xsd": "XSD",
   "dashboard.data_models": "Datamodeller",
   "dashboard.description": "Beskrivelse",
-  "dashboard.edit_app": "Rediger appen {{appName}}",
-  "dashboard.edit_app_icon": "Rediger app",
-  "dashboard.edit_data_models": "Rediger modeller",
-  "dashboard.edit_service": "Rediger app",
+  "dashboard.edit_app": "Rediger {{appName}} i Studio",
+  "dashboard.edit_datamodels": "Rediger {{appName}} i Studio",
```

Comment:
`"dashboard.edit_datamodels"` should still be: `"dashboard.edit_data_models"`

Reply:
Fixed ✅